src/main/java/ch/njol/skript/effects/EffPathfind.java

```diff
@@ -72,7 +72,7 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 	@Override
 	protected void execute(Event event) {
 		Object target = this.target != null ? this.target.getSingle(event) : null;
-		int speed = this.speed != null ? this.speed.getSingle(event).intValue() : 1;
+		double speed = this.speed != null ? this.speed.getSingle(event).doubleValue() : 1;
```

Comment:
You need to make sure that getSingle() doesn't return null before calling doubleValue()

Reply:
It won't